src/clr-core/alert/alert-actions.element.ts

```diff
@@ -24,10 +24,9 @@ import { styles } from './alert-actions.element.css';
  * </cds-app-alert>
  * ```
  *
- * @beta 3.0
- * @element cds-alert-actions
+ * @beta
  */
-// @dynamic
+@customElement('cds-alert-actions')
```

Comment:
Same API as lit-element but uses our registration code under the hood.

Reply:
We had trouble in the past with Lit's `customElement` decorator. That's why we stopped using it. Are those issues no longer a problem for us?

Reply:
The issue with the lit element decorator is that it doesnt check if the element has already been registered so it will throw an exception if imported more than once. This decorator is the same api but instead of registering directly it calls our safe register func under the hood.